addons/cloud-games/addon.json

```diff
@@ -10,6 +9,11 @@
       "name": "griffpatch"
     }
   ],
+  "popup": {
+    "addonId": "cloud-games",
```

Comment:
Seems unnecessary.

Reply:
Including the id? I guess so... I guess I'll code a bit more to remove it.